src/components/product/ProductCard.component.tsx

```diff
@@ -9,12 +14,26 @@ interface ProductCardProps {
 }
 
 const ProductCard: React.FC<ProductCardProps> = ({ product }) => {
+    const { isError, isLoading } = useContext(ProductsDataContext);
+
     const navigate = useNavigate();
 
     const showProductPage = () => {
```

Comment:
If you don't have a product page, proly you should remove it.
![image](https://github.com/demasha0908/fe-react-2024/assets/98031768/775f6df0-5c1a-4386-bcff-1d2aae29d1bf)

Reply:
Fixed